content/docs/getting-started/integrations/gitlab.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: GitLab CI
+description: |
+  Learn how to build and deploy your unikernel application via GitLab CI.
+---
+
+## Integrating Kraft in GitLab CI
+
+Using `kraft` inside the GitLab CI system is no different from any other command and can be used for all its use cases:
+
+* Build Unikraft unikernels;
+* Run Unikraft unikernels in emulation mode;
+* Package and push OCI images to compatible registries.
+
+<Info>
+Get started quickly and easily by using the [sample GitLab repository](https://gitlab.com/unikraft/gitlab-ci-example).
+</Info>
+
+### Workflow Steps
+
+We break up the resulting workflow into multiple steps to guide people through the sytax of GitLab, its specifics, and how they relate to KraftKit.
+
+#### 1. Setting Variables
+
+The first and foremost thing we must do is set the general KraftKit environmental variables.
+
+```yaml
+variables:
+  KRAFTKIT_LOG_LEVEL: debug
+  KRAFTKIT_LOG_TYPE: basic
+  KRAFTKIT_NO_CHECK_UPDATES: "true"
+```
+
+Logging we need to set to 'basic' to ensure that the output is not mangled and the level we can pick from the following list: `debug`/`info`/`warn`/`error`.
+We pick `debug` to make sure we do not miss anything important in our workflows.
+Finally, `KRAFTKIT_NO_CHECK_UPDATES` we set for redundancy, especially in the case where we might use an older version of KraftKit.
+
+#### 2. Using the Correct Image
+
+The default image for KraftKit in CI/CD is `kraftkit.sh/base`, but this is not fully compatible with GitLab.
```

Comment:
`kraftkit.sh/base:latest`

Reply:
Why is it not compatible with GitLab?

Reply:
What's the fix to make it work with GitLab? Could we make this fix before writing this in the docs?